src/implementations/twig/components/datepicker/datepicker.story.js

```diff
@@ -1,45 +1,44 @@
 import { withNotes } from '@ecl/storybook-addon-notes';
-import { withKnobs, text, boolean, optionsKnob } from '@storybook/addon-knobs';
 import withCode from '@ecl/storybook-addon-code';
-import { getExtraKnobs, tabLabels, getFormKnobs } from '@ecl/story-utils';
+import { getFormControls, correctSvgPath } from '@ecl/story-utils';
 
-import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import dataDefault from '@ecl/specs-component-datepicker/demo/data';
 import datepicker from './datepicker.html.twig';
 import notes from './README.md';
 
-const prepareDatePicker = (data) => {
-  getFormKnobs(data);
-  data.autoinit = boolean('autoinit', data.autoinit, tabLabels.states);
-  data.default_value = text(
-    'default_value',
-    data.default_value,
-    tabLabels.optional
-  );
-  data.label = text('label', data.label, tabLabels.required);
-  data.icons_path = optionsKnob(
-    'icons_path',
-    { current: defaultSprite, 'no path': '' },
-    defaultSprite,
-    { display: 'inline-radio' },
-    tabLabels.required
-  );
-
-  data.placeholder = text('placeholder', data.placeholder, tabLabels.required);
-
-  getExtraKnobs(data);
+const getArgTypes = (data) => {
+  return {
+    ...getFormControls(data),
+    autoinit: {
```

Comment:
Do we really need to have autoinit control ?

Reply:
i think so because this is the only component that has this opt-in for the autoinit, i think it's important to demo it